useful-resources/privacy-policy.md

```diff
@@ -36,115 +24,192 @@ I dati personali raccolti rispettano il principio generale di "minimizzazione de
 - Residenza
 - Indirizzo email
 
```

Comment:
Add `### Sicurezza sui luoghi di lavoro`
- Nome
- Cognome
- Data di nascita
- Codice fiscale
- Numero di telefono
- Residenza/Domicilio

Reply:
Also, add `Software di gestione amministrativa`
- Nome
- Cognome
- email con dominio @nebulab.it